Scripts/Mac/BuildGVFSForMac.sh

```diff
@@ -67,7 +67,6 @@ echo 'Copying native binaries to Publish directory...'
 cp $VFS_OUTPUTDIR/GVFS.Native.Mac/Build/Products/$CONFIGURATION/GVFS.ReadObjectHook $VFS_PUBLISHDIR || exit 1
 cp $VFS_OUTPUTDIR/GVFS.Native.Mac/Build/Products/$CONFIGURATION/GVFS.VirtualFileSystemHook $VFS_PUBLISHDIR || exit 1
 cp $VFS_OUTPUTDIR/GVFS.Native.Mac/Build/Products/$CONFIGURATION/GVFS.PostIndexChangedHook $VFS_PUBLISHDIR || exit 1
-cp -Rf $VFS_OUTPUTDIR/GVFS.Notifications/VFSForGit.Mac/Build/Products/$CONFIGURATION/VFS\ For\ Git.app $VFS_PUBLISHDIR || exit 1
```

Comment:
Why was this line removed?

Reply:
We were installing `VFS For Git.app` redundantly in two different directories - 1. `/Library/Application Support/VFS For Git` and 2. `/usr/local/vfsforgit`. This is not required and seems to cause weird installer issues during testing. I've already removed this in pending [PR](https://github.com/microsoft/VFSForGit/pull/1202). Just wanted to see if it helps @jamill with the weird `VFS For Git.app` permissions issue that he is seeing.